platform/platform-app/src/main/resources/features.xml

```diff
@@ -1025,6 +1027,25 @@
         <feature>util-uuidgenerator</feature>
     </feature>
 
+    <feature name="kernel" install="manual" version="${project.version}">
+        <feature prerequisite="true" version="${karaf.version}">standard</feature>
+        <feature prerequisite="true" version="${karaf.version}">eventadmin</feature>
+        <feature prerequisite="true" version="${karaf.version}">jasypt-encryption</feature>
+        <feature prerequisite="true" version="${karaf.version}">http</feature>
+    </feature>
+
+    <feature name="branding-api" version="${project.version}" description="Branding API">
+        <bundle>mvn:ddf.platform/branding-api/${project.version}</bundle>
+    </feature>
+
+    <feature name="ddf-branding" version="${project.version}" description="DDF Branding">
+        <feature prerequisite="true">web-container</feature>
```

Comment:
@vinamartin and I will handle investigating removing the prereqs here in [DDF-3008](https://codice.atlassian.net/browse/DDF-3008).